pkg/operator/ceph/disruption/clusterdisruption/reconcile.go

```diff
@@ -190,12 +190,7 @@ func (r *ReconcileClusterDisruption) reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, err
 	}
 
-	disabledPDB, ok := pdbStateMap.Data[drainingFailureDomainKey]
-	if ok && len(disabledPDB) > 0 {
-		return reconcile.Result{Requeue: true, RequeueAfter: 30 * time.Second}, nil
-	}
-
-	return reconcile.Result{}, nil
+	return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
```

Comment:
Questions:
- How about if we make the 30s configurable in the cluster CR?
- Is this reconcile always guaranteed to requeue? Or what happens in case of errors above where we don't requeue? How will the reconcile be triggered again?

Reply:
For the second question, reconcile will requeue every 30 seconds. The Controller will requeue the Request to be processed again if the returned error is non-nil or Result.Requeue is true